packages/zapp/console/src/components/Literals/test/helpers/literalHelpers.ts

```diff
@@ -0,0 +1,80 @@
+import { Core, Protobuf } from 'flyteidl';
+
+export function getIValue(
+  kind: 'nullValue' | 'numberValue' | 'stringValue' | 'boolValue' | 'structValue' | 'listValue',
+  value?:
+    | Protobuf.NullValue
+    | number
+    | string
+    | boolean
+    | Protobuf.IStruct
+    | Protobuf.IListValue
+    | null,
+): Core.Primitive | Core.IPrimitive {
+  return {
+    kind,
+    [kind]: value,
+  } as any as Protobuf.Value;
+}
+
+export function getPrimitive(
+  key: 'integer' | 'floatValue' | 'stringValue' | 'boolean' | 'datetime' | 'duration',
+  value?: Long | number | string | boolean | Protobuf.ITimestamp | Protobuf.IDuration | null,
+): Core.Primitive | Core.IPrimitive {
+  return {
+    [key]: value,
+    value: key,
+  } as any as Core.Primitive;
+}
+
+export function generateBlobType(
+  format?: string,
+  dimensionality?: Core.BlobType.BlobDimensionality,
+  uri?: string,
+): Core.IBlob {
+  return {
+    metadata: {
+      type: {
+        format,
+        dimensionality,
+      },
+    },
+    uri,
+  };
+}
+
+// TOP LEVEL SCHEMA GENERATORS:
+export const getScalar = (
+  value: Core.IPrimitive | Core.IBlob | Core.IBinary,
+  scalarType: string,
+) => {
+  return {
+    scalar: {
+      [scalarType]: value,
+      value: scalarType,
+    },
+    value: 'scalar',
+  } as Core.IScalar;
+};
+
+export const getCollection = (literals: Core.ILiteral[]) => {
+  return {
+    result_var: {
+      collection: {
+        literals,
+      },
+      value: 'collection',
+    } as Core.ILiteralCollection,
+  };
+};
```

Comment:
nit: Ideally we should avoid any `as` casting, as it skips TS type checking.
Ideally in such cases we should do something like this:
```
  const resultVar: Core.ILiteralCollection = {
    collection: {
      literals,
    },
    value: 'collection',
  };
  return {
    result_var: resultVar,
  };
```
but than we can see that collection doesn't exist on , as it has:
```
        interface ILiteralCollection {
            /** LiteralCollection literals */
            literals?: (flyteidl.core.ILiteral[]|null);
        }
```
So should we use some other type here?